src/main/java/com/cburch/logisim/std/io/extra/Buzzer.java

```diff
@@ -278,17 +285,20 @@ public void removeComponent(Circuit circ, Component c, CircuitState state) {
   }
 
   private interface BuzzerWaveformStrategy {
-    double amplitude(int i, double hz, double pw);
+    double amplitude(double i, double hz, double pw);
   }
 
   private static class Data implements InstanceData {
 
-    private static final int SAMPLE_RATE = 44100;
+    private int sampleRate = 12345;
```

Comment:
Why is `sampleRate` initialised with this magic constant? If it is overwritten later anyway, why not initialising it at all (giving a static analyser the chance to report missed paths) or to zero?

Reply:
Indeed, I put that value there for testing purposes and completely forgot to remove it, I'll do that in a minute

Reply:
Thanks @zdimension!